containers/Ideas/__tests__/IdeaProject.test.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import '@testing-library/jest-native/extend-expect';
+import { render, fireEvent, act } from '@testing-library/react-native';
+import { IdeaProject } from '../IdeaProject';
+import API from '../../../BaseApi';
+
+const promise = Promise.resolve();
+jest.mock('../../../BaseApi', () => jest.fn());
+API.mockImplementation(() => ({
+  __esModule: true,
+  default: {
+    getModule: () => promise
+  }
+}));
+
+jest.mock('../../../components/Richtext', () => ({
+  __esModule: true,
+  Richtext: () => (<></>)
+}));
+
+test('Contact Infos', async () => {
+  const mockProject = {
+    route: {
+      params: {
+        project: {
+          has_contact_info: true,
+          contact_name: 'some name',
+          contact_address_text: 'some address',
+          contact_phone: '1234',
+          contact_email: 'some@email.com',
+          contact_url: 'someurl.com',
+        }
+      },
+    },
+    navigation: {
+      addListener: jest.fn(),
+      goBack: jest.fn(),
+      navigate: jest.fn()
+    }
+  };
+
+  const tree = render(<IdeaProject {...mockProject} />);
+  const infoTabBtn = tree.getByTestId('info-tab-button');
```

Comment:
using `getText('Information')` works fine for me, so I think we should remove the `testID`